pkgs/applications/misc/clipqr/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitLab
+, buildGoModule
+, copyDesktopItems
+, makeDesktopItem
+, pkg-config
+, pkgs
```

Comment:
This shouldn't be done. You need to specify each input here. X11 libraries should be specified separately (i.e. **not** with the `xorg` attribute).
```suggestion
```